test/content-parser.test.js

```diff
@@ -10,43 +9,45 @@ const first = function (req, payload, done) {}
 const second = function (req, payload, done) {}
 const third = function (req, payload, done) {}
 
-test('hasContentTypeParser', t => {
-  test('should know about internal parsers', t => {
+test('hasContentTypeParser', async t => {
```

Comment:
should we not use `describe` here?

Reply:
I changed 50+ files until now as part of the migration from `tap` to `node:test` and didn't use `describe`, only tests, and sub tests, since I thought it's equivalent for the tests before using `tap`.